shared/src/tx_pool/pool.rs

```diff
@@ -158,6 +158,14 @@ impl TxPool {
             || self.orphan.contains_key(id)
     }
 
+    pub fn contains_tx(&self, id: &ProposalShortId) -> bool {
+        self.pending.contains_key(id)
+            || self.gap.contains_key(id)
+            || self.proposed.contains_key(id)
+            || self.orphan.contains_key(id)
+            || self.conflict.contains_key(id)
```

Comment:
I think the `conflict pool` should not be added to the judgment. The tx in the conflict pool may not be updated for a long time, there will be dirty data